PickUpApi/Infrastructure/db/migrations/20210831180807-create-restaurant.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('restaurants', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      Name: {
+        type: Sequelize.STRING
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
```

Comment:
I thought that the restaurant should have a longitude and latitude fields ? or are we not considering those anymore ?